Lib/test/test_pkgutil.py

```diff
@@ -595,18 +590,28 @@ def test_get_loader_None_in_sys_modules(self):
             del sys.modules[name]
         self.assertIsNone(loader)
 
+    def test_get_loader_is_deprecated(self):
+        for module in ["sys", "os", "test.support"]:
```

Comment:
Why the specific check for multiple modules? Any reason to not just import any module and check the warning appears? Same with below.

Reply:
Old tests used to have these lines:

```python
self.assertIsNotNone(pkgutil.find_loader("sys"))       self.assertIsNotNone(pkgutil.find_loader("os"))      self.assertIsNotNone(pkgutil.find_loader("test.support"))
```

So, I've copied the same behaviour.
Will address your review soon1